packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -50,6 +51,7 @@
                 <component
                     :is="titleHtmlTag"
                     v-if="title"
+                    id="modal-title"
```

Comment:
I know it shouldn't happen, but I've seen it before, sometimes two modals can be displayed at once which would mean there could be duplicate ids in the page markup that I think would break this?

Reply:
ah good point, thanks @DamianMullins! Just thinking about how else to go about it and @FayeCarter just sent me this which could be a good work around - https://github.com/justeat/fozzie-components/blob/master/packages/components/atoms/f-link/src/components/Link.vue#L79 👏